administration/users.rst

```diff
@@ -8,6 +8,7 @@ However, |omv| also maintains control over these users, **so management is a tea
 between the Debian operating system and the internal database of |omv|.
 
 Users that are managed via the |webui| are so-called *non-system users*.
+Identification is managed by the *UID* and groups by the *GID*.
```

Comment:
This is IMP incorrect. What about the name which is also unique?
Second, this information is not useful for the user here.

Please remove because of the above points.

Reply:
Ok, after some time i understood the meaning of the sentence. As the content wasn't really obvious straight away, I changed it.

Reply:
XD